test/metabase/api/card_test.clj

```diff
@@ -281,10 +281,11 @@
                         card-3 :rasta]
         (with-cards-in-readable-collection [card-1 card-2 card-3 card-4]
           (testing "\nShould return cards that were recently viewed by current user only"
-            (is (= ["Card 3"
-                    "Card 4"
-                    "Card 1"]
-                   (map :name (mt/user-http-request :rasta :get 200 "card", :f :recent))))))))))
+            (let [recent-card-names (->> (mt/user-http-request :rasta :get 200 "card", :f :recent)
+                                         (map :name)
+                                         (filter #{"Card 1" "Card 2" "Card 3" "Card 4"}))]
+              (is (= ["Card 3" "Card 4" "Card 1"]
```

Comment:
This is more concisely expressed with `(is (contains? ` and you could do without the `filter`.

Reply:
I had something like that at first, but I think preserving the order (3→4→1) is worthwhile.

Reply:
Ah, yes. Sorry. I remember you writing a test helper for this once. Something like this

```clojure
(defn contains-sequence [seq1 seq2]
  (loop [remaining-seq1 seq1
         remaining-seq2 seq2]
    (cond
      (empty? remaining-seq2) true
      (empty? remaining-seq1) false
      (= (first remaining-seq1) (first remaining-seq2))
      (recur (rest remaining-seq1) (rest remaining-seq2))
      :else (recur (rest remaining-seq1) seq2))))
```